apps/extension/src/extension/ExtensionRequester.ts

```diff
@@ -59,4 +65,8 @@ export class ExtensionRequester {
 
     return result.return;
   }
+
+  startSession(): void {
```

Comment:
We call `startSession()` from several places where we want to post any messages using requester. Could it be an issue or leave to unintended behavior if several sessions are "started" in parallel? I see it ends up calling this

```ts
const SESSION_PORT = "session-port";
browser.runtime.connect({
  name: SESSION_PORT,
});
```